x/fungible/module.go

```diff
@@ -178,6 +178,14 @@ func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
 			ctx.Logger().Error("Unable To deploy contracts", "err", err.Error())
 		}
 	}
+	if ctx.BlockHeight() == 20 {
+		err := am.keeper.UpdateSystemContractAddress(sdk.WrapSDKContext(ctx))
+		if err != nil {
+			ctx.Logger().Error("Unable To update system contracts", "err", err.Error())
+		} else {
+			ctx.Logger().Info("System contracts updated")
+		}
+	}
```

Comment:
I don't understand this part, why doing it in `BeginBlock` at height 20?

Reply:
Just to test the Msg handler without actually making a tx in smoketest. 
This function UpdateSystemContractAddress() is empty in builds other than PRIVNET build.